networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/versions/GossipForkSubscriptionsDeneb.java

```diff
@@ -101,9 +110,24 @@ protected void addGossipManagers(final ForkInfo forkInfo) {
     addSignedBlsToExecutionChangeGossipManager(forkInfo);
 
     // Deneb
+    addBlobSidecarGossipManager(forkInfo);
     addBlockAndBlobsSidecarGossipManager(forkInfo);
   }
 
+  void addBlobSidecarGossipManager(final ForkInfo forkInfo) {
+    blobSidecarGossipManager =
+        new BlobSidecarGossipManager(
+            recentChainData,
+            spec,
+            asyncRunner,
+            discoveryNetwork,
+            gossipEncoding,
+            forkInfo,
+            blobSidecarProcessor,
+            getMessageMaxSize());
```

Comment:
we should double check which size we should apply here

Reply:
I had a look and GOSSIP_MAX_SIZE_BELLATRIX and MAX_CHUNK_SIZE_BELLATRIX haven't been changed yet for Deneb